app/controllers/api/v1/base_controller.rb

```diff
@@ -7,8 +7,6 @@ class API::V1::BaseController < ApplicationController
 
   before_action :authenticate
 
-  serialization_scope :view_context
```

Comment:
This doesn't break any specs; I don't quite know what `view_context` does here. Old docs for reference: https://github.com/rails-api/active_model_serializers/tree/0-9-stable#customizing-scope